client/src/components/QueryComponent/QueryComponent.js

```diff
@@ -31,17 +31,15 @@ class QueryComponent extends Component {
     this.state = {
       queries: [],
       selectedQuery: defaultQuery,
-      selectedQueryMultiline: "",
+      selectedQueryHold: defaultQuery,
```

Comment:
Maybe `selectedQueryWithNewlines` or similar would be a better name for this state property?

Reply:
Done